interface/wc_decision_icons.gfx

```diff
@@ -348,4 +348,8 @@ spriteTypes = {
 		name = "GFX_repay_money"
 		texturefile = "gfx\\interface\\decision_icons\\repay_action.tga"
 	}
+	spriteType = {
+		name = "GFX_create_blood_knights_order"
+		texturefile = "gfx\\interface\\decision_icons\\restore_the_sunwell.dds"
```

Comment:
Can we get a new icon?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81817032-271d5980-953d-11ea-80b2-4e21a11375b6.png)
![image](https://user-images.githubusercontent.com/46384992/81817053-2edcfe00-953d-11ea-86b1-ed076805329d.png)

</details>

Reply:
Will try to find it then and make it .dds file